services/src/modules/arguments-injection/arguments-injection.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { RequestContext } from '../context';
+import { inject } from './arguments-injection';
+
+interface TestCase {
+  template: string;
+  source?: unknown;
+  args?: Record<string, unknown>;
+  context?: RequestContext;
+  expected: unknown;
+}
+
+const testCases: [string, TestCase][] = [
```

Comment:
Probably worth adding a test case for `jwt` since it has special handling (being taken from the `context`). Also possibly for `exports` (though that one might be tricky)